packages/shared/src/modules/activity-logs/libs/types/activity-log-query-parameters.type.ts

```diff
@@ -1,5 +1,6 @@
 type ActivityLogQueryParameters = {
 	endDate: string;
+	hasHidden?: boolean;
```

Comment:
I meant to only pass this parameter as true on activity log service level, similarly, we did for hasDeleted parameter, where we needed to find with deleted users in one place and without them in another place. And don't we need to pass this parameter to findAllByProjectId?

Reply:
Oops, now I see it clearly — it’s much simpler now. Regarding `findAllByProjectId`, I added the same check as I did for `findAllWithoutPagination`, but I already had a check in place that handles this: `.andWhere("contributors.isHidden", false)`.

After you pointed out that this should be handled on the backend, I tried implementing it the way I originally thought one more time. Surprisingly (maybe Visual Studio was bugging earlier), it's now working without adding hasHidden at all, just using the `.where("contributors.isHidden", false)` condition.

However, the new solution might be more consistent with the project's structure, similar to hasDeleted, as you mentioned.